drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -1,5 +1,35 @@
 zigbeeManufacturer:
   #HANSSEM SMART SWITCH
```

Comment:
this comment is for the existing fingerprint below

your fps can go above this comment